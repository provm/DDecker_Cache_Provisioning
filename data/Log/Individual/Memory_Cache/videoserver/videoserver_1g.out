========== Container filebench Memory STATS ==============
cache 23474176
rss 7589888
rss_huge 0
mapped_file 8032256
writeback 0
swap 0
pgpgin 77229
pgpgout 69645
pgfault 172288
pgmajfault 195
inactive_anon 49152
active_anon 7565312
inactive_file 10850304
active_file 12537856
unevictable 0
hierarchical_memory_limit 999997440
hierarchical_memsw_limit 9223372036854771712
total_cache 23474176
total_rss 7589888
total_rss_huge 0
total_mapped_file 8032256
total_writeback 0
total_swap 0
total_pgpgin 77229
total_pgpgout 69645
total_pgfault 172288
total_pgmajfault 195
total_inactive_anon 49152
total_active_anon 7565312
total_inactive_file 10850304
total_active_file 12537856
total_unevictable 0
======== END Container STATS ======
WARNING: Could not open /proc/sys/kernel/shmmax file!
It means that you probably ran Filebench not as a root. Filebench will not increase shared
region limits in this case, which can lead to the failures on certain workloads.
Filebench Version 1.4.9.1
 1852: 0.000: Allocated 170MB of shared memory
 1852: 0.009: Eventgen rate taken from variable
 1852: 0.010: Video Server Version 3.0 personality successfully loaded
 1852: 0.010: Creating/pre-allocating files and filesets
 1852: 0.035: Fileset passivevids: 10 files, 0 leafdirs, avg dir width = 20, avg dir depth = 0.8, 5699.813MB
 1852: 0.035: Re-using fileset passivevids.
 1852: 0.035: Creating fileset passivevids...
 1852: 0.035: Preallocated 6 of 10 of fileset passivevids in 1 seconds
 1852: 0.035: Fileset activevids: 5 files, 0 leafdirs, avg dir width = 4, avg dir depth = 1.2, 7474.542MB
 1852: 0.035: Re-using fileset activevids.
 1852: 0.035: Creating fileset activevids...
 1852: 0.050: Preallocated 5 of 5 of fileset activevids in 1 seconds
 1852: 0.050: waiting for fileset pre-allocation to finish
 1876: 32.099: Starting 1 vidreaders instances
 1876: 32.332: Starting 1 vidwriter instances
 1878: 32.503: Starting 1 vidwriter threads
 1877: 32.831: Starting 16 vidreaders threads
 1852: 33.848: Running...
