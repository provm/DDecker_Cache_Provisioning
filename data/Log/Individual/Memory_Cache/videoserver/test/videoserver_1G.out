========== Container filebench Memory STATS ==============
cache 23973888
rss 7577600
rss_huge 0
mapped_file 8032256
writeback 0
swap 0
pgpgin 77723
pgpgout 70020
pgfault 173090
pgmajfault 195
inactive_anon 49152
active_anon 7569408
inactive_file 10936320
active_file 12947456
unevictable 0
hierarchical_memory_limit 999997440
hierarchical_memsw_limit 9223372036854771712
total_cache 23973888
total_rss 7577600
total_rss_huge 0
total_mapped_file 8032256
total_writeback 0
total_swap 0
total_pgpgin 77723
total_pgpgout 70020
total_pgfault 173090
total_pgmajfault 195
total_inactive_anon 49152
total_active_anon 7569408
total_inactive_file 10936320
total_active_file 12947456
total_unevictable 0
======== END Container STATS ======
WARNING: Could not open /proc/sys/kernel/shmmax file!
It means that you probably ran Filebench not as a root. Filebench will not increase shared
region limits in this case, which can lead to the failures on certain workloads.
Filebench Version 1.4.9.1
 1851: 0.000: Allocated 170MB of shared memory
 1851: 0.002: Eventgen rate taken from variable
 1851: 0.002: Video Server Version 3.0 personality successfully loaded
 1851: 0.002: Creating/pre-allocating files and filesets
 1851: 0.022: Fileset passivevids: 10 files, 0 leafdirs, avg dir width = 20, avg dir depth = 0.8, 5699.813MB
 1851: 0.022: Re-using fileset passivevids.
 1851: 0.022: Creating fileset passivevids...
 1851: 0.022: Preallocated 6 of 10 of fileset passivevids in 1 seconds
 1851: 0.022: Fileset activevids: 5 files, 0 leafdirs, avg dir width = 4, avg dir depth = 1.2, 7474.542MB
 1851: 0.022: Re-using fileset activevids.
 1851: 0.022: Creating fileset activevids...
 1851: 0.041: Preallocated 5 of 5 of fileset activevids in 1 seconds
 1851: 0.041: waiting for fileset pre-allocation to finish
 1880: 30.993: Starting 1 vidreaders instances
 1880: 33.346: Starting 1 vidwriter instances
 1882: 34.421: Starting 1 vidwriter threads
 1881: 34.457: Starting 16 vidreaders threads
 1851: 36.474: Running...
 1851: 1836.725: Run took 1800 seconds...
 1851: 1837.000: Per-Operation Breakdown
vidreader            15359531ops     8531ops/s 2130.8mb/s      1.6ms/op     1352us/op-cpu [0ms - 1585ms]
replaceinterval      71ops        0ops/s   0.0mb/s  20000.2ms/op      141us/op-cpu [20000ms - 20003ms]
wrtclose             72ops        0ops/s   0.0mb/s      0.0ms/op        0us/op-cpu [0ms - 0ms]
newvid               72ops        0ops/s  22.5mb/s   4910.7ms/op   736806us/op-cpu [135ms - 23248ms]
wrtopen              72ops        0ops/s   0.0mb/s      1.2ms/op      417us/op-cpu [0ms - 46ms]
vidremover           72ops        0ops/s   0.0mb/s    193.4ms/op     4306us/op-cpu [0ms - 7447ms]
 1851: 1837.000: IO Summary: 15359819 ops, 8531.267 ops/s, (8531/0 r/w), 2153.3mb/s,     95us cpu/op,   1.6ms latency
 1851: 1837.000: Shutting down processes
========== Container 1 Memory STATS AT END ==============
cache 830308352
rss 962560
rss_huge 0
mapped_file 2093056
writeback 0
swap 5652480
pgpgin 55666162
pgpgout 55463215
pgfault 30990786
pgmajfault 2270
inactive_anon 851968
active_anon 110592
inactive_file 198250496
active_file 632057856
unevictable 0
hierarchical_memory_limit 999997440
hierarchical_memsw_limit 9223372036854771712
total_cache 830308352
total_rss 962560
total_rss_huge 0
total_mapped_file 2093056
total_writeback 0
total_swap 5652480
total_pgpgin 55666162
total_pgpgout 55463215
total_pgfault 30990786
total_pgmajfault 2270
total_inactive_anon 851968
total_active_anon 110592
total_inactive_file 198250496
total_active_file 632057856
total_unevictable 0
======== END Container STATS ======
